arm/cookbooks/motd/.kitchen.local.azure.example.yml

```diff
@@ -0,0 +1,26 @@
+# 1 rename this to .kitchen.local.yml
+# 2 update the value for your env
+---
+driver:
+  name: azurerm
+
+driver_config:
+  subscription_id: '<%= ENV['SUBSCRIPTION_ID'] %>'
+  location: 'South Central US'
+  machine_size: 'Standard_D1'
+
+platforms:
+  - name: windows2012-r2
+    driver_config:
+      image_urn: MicrosoftWindowsServer:WindowsServer:2012-R2-Datacenter:latest
+    transport:
+      name: winrm
+
+suites:
+  - name: default
+    run_list:
+      - recipe[motd::default]
+    verifier:
+      inspec_tests:
+        - test/recipes
+        - spec/unit/recipes/default_spec.rb
```

Comment:
Ah -- this is any easy mistake to make. The tests in `spec/unit/...` don't have anything to do with test-kitchen. They are only for use with ChefSpec (think RSpec) unit tests while the stuff in `test/recipes` are the kitchen integration tests (think Selenium).  

So the test for Chef cookbooks are:
1. lexical/style (rubocop, foodcritic)
2. unit tests (rspec w/ ChefSpec)
3. integration tests (test-kitchen)

In short, just delete line 26....